include/onnxruntime/core/session/onnxruntime_run_options_config_keys.h

```diff
@@ -25,3 +25,8 @@
 // Example usage: "cpu:0;gpu:0" (or) "gpu:0"
 // By default, the value for this key is empty (i.e.) no memory arenas are shrunk
 static const char* const kOrtRunOptionsConfigEnableMemoryArenaShrinkage = "memory.enable_memory_arena_shrinkage";
+
+// when any CUDA work is issued (CUDA + TensorRT EP) the InferenceSession::run method will synchronize
```

Comment:
It's not necessarily specific to CUDA. Let's keep the original comment from run_options.h.

Reply:
Done but i extended it to mention the default value. Is there a way to set the default value globally for those options?